linkis-engineconn-plugins/hive/src/main/scala/org/apache/linkis/engineplugin/hive/executor/HiveEngineConnExecutor.scala

```diff
@@ -205,7 +205,15 @@ class HiveEngineConnExecutor(
       val startTime = System.currentTimeMillis()
       try {
         Utils.tryCatch {
-          val queryPlan = driver.getPlan
+          var queryPlan = driver.getPlan
```

Comment:
This problem is caused by another fix. It is recommended to optimize and add parameters, whether to enable compilation.#3906
```
eg:
val ENABLE_HIVE_COMPILE =
    CommonVars[Boolean]("linkis.hive.enable.compile", true).getValue
```
![image](https://user-images.githubusercontent.com/11496700/218317978-671c0b4c-5ad8-4183-af26-f3b2d45b7f7f.png)

Reply:
> This problem is caused by another fix. It is recommended to optimize and add parameters, whether to enable compilation.#3906
> 
> ```
> eg:
> val ENABLE_HIVE_COMPILE =
>     CommonVars[Boolean]("linkis.hive.enable.compile", true).getValue
> ```
> 
> ![image](https://user-images.githubusercontent.com/11496700/218317978-671c0b4c-5ad8-4183-af26-f3b2d45b7f7f.png)

Thanks for your kind advice. I will optimize and add parameters, whether to enable compilation.

Reply:
> 

I have modify the code according to our talk, here is a brief explanation of code:
1, Compile the code.
2, Invoke driver.getPlan to get execution plan.
3, Invoke driver.run with parameter not compile ,avoiding recompile the query.

Reply:
ok， Thanks a lot.